src/jorphan/src/main/java/org/apache/commons/cli/avalon/CLOptionDescriptor.java

```diff
@@ -177,23 +168,9 @@ public final String getName() {
         return this.name;
     }
 
-    /**
-     * Convert to String.
-     *
-     * @return the converted value to string.
-     */
     @Override
     public final String toString() {
-        final StringBuilder sb = new StringBuilder();
-        sb.append("[OptionDescriptor ");
-        sb.append(this.name);
-        sb.append(", ");
-        sb.append(this.id);
-        sb.append(", ");
-        sb.append(this.flags);
-        sb.append(", ");
-        sb.append(this.description);
-        sb.append(" ]");
-        return sb.toString();
+        return String.format("[OptionDescriptor %s, %d, %d, %s ]",
```

Comment:
@ham1 , would you please refrain from `String#format`?
Technically speaking, it uses regexps behind the scene, and string+string would look simpler in this case

Reply:
I did try to avoid it where I thought performance mattered, but I think this is more readable and not performance critical section.
I think:

```
String.format("[OptionDescriptor %s, %d, %d, %s ]", 
        this.name, this.id, this.flags, this.description);
```
is slightly more readable than:
```
"[OptionDescriptor " + this.name + ", " + this.id + ", " + this.flags + ", + this.description + " ]";
```

Happy to do the latter if required and either way we should probably remove that space before the last `]`.

Reply:
Just in case, I don't think `format` helps readability.
For instance, it is easy to mess up with `%s %d %s` vs `%d %s`, and it is hard to trace the order of the arguments.

`"[OptionDescriptor " + name + ", " + id + ", " + flags + ", + description + "]"`

is just fine.

Reply:
I've applied your suggestion, removing this helps!

Reply:
my fear was "would Checkstyle accept that?" :)